python/ray/data/_internal/planner/plan_download_op.py

```diff
@@ -86,7 +87,7 @@ def plan_download_op(
             partition_map_transformer,
             input_physical_dag,
             data_context,
-            name="URIPartitioner",
+            name=f"Partition[{uri_column_names_str}]",
```

Comment:
Here and for Download -- replace brackets (`[`) with parens (`(`) or consistency with how we name map operators?
```suggestion
            name=f"Partition({uri_column_names_str})",
```

Reply:
done